api-samples/printing/printers.js

```diff
@@ -6,62 +6,75 @@ function onPrintButtonClicked(printerId, dpi) {
   var ticket = {
     version: '1.0',
     print: {
-      color: { type: 'STANDARD_MONOCHROME' },
-      duplex: { type: 'NO_DUPLEX' },
-      page_orientation: { type: 'LANDSCAPE' },
-      copies: { copies: 1 },
-      dpi: {
-        horizontal_dpi: dpi.horizontal_dpi,
-        vertical_dpi: dpi.vertical_dpi
-      },
+      color: {type: 'STANDARD_MONOCHROME'},
+      duplex: {type: 'NO_DUPLEX'},
+      page_orientation: {type: 'LANDSCAPE'},
+      copies: {copies: 1},
+      dpi: {horizontal_dpi: dpi.horizontal_dpi, vertical_dpi: dpi.vertical_dpi},
       media_size: {
         width_microns: 210000,
         height_microns: 297000,
         vendor_id: 'iso_a4_210x297mm'
       },
-      collate: { collate: false }
+      collate: {collate: false}
     }
   };
 
   fetch('test.pdf')
-    .then((response) => response.arrayBuffer())
-    .then((arrayBuffer) => {
-      const request = {
-        job: {
-          printerId: printerId,
-          title: 'test job',
-          ticket: ticket,
-          contentType: 'application/pdf',
-          document: new Blob([new Uint8Array(arrayBuffer)], {
-            type: 'application/pdf'
-          })
-        }
-      };
-      chrome.printing.submitJob(request, (response) => {
-        if (response !== undefined) {
-          console.log(response.status);
-        }
-        if (chrome.runtime.lastError !== undefined) {
-          console.log(chrome.runtime.lastError.message);
-        }
+      .then(response => response.arrayBuffer())
```

Comment:
`eslint --fix` will take care of this automagically, but a style nit would be to always or never use prens with a single argument (the corp style is to always use them)